clang/test/Driver/sycl-use-int-headers.cpp

```diff
@@ -0,0 +1,41 @@
+// This test tests the option -fsycl-use-integration-headers
+
+// Ensure that by default the -fsycl-use-integration-headers gets passed to the cc1 invocation.
+// RUN: %clang -### -c -fsycl %s 2>&1 | FileCheck %s -check-prefix=CHECK-DEFAULT
+// CHECK-DEFAULT: "-fsycl-use-integration-headers" "-D__INTEL_SYCL_USE_INTEGRATION_HEADERS"
```

Comment:
Perhaps also worth testing:
1. Check default behavior with `-fsycl-device-only`.
2. Check that neither `-fsycl-use-integration-headers` nor `-fno-sycl-use-integration-headers` is passed when neither of `-fsycl` or `-fsycl-device-only` is passed.

Reply:
Thanks, done.